atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/LockWatchValueScopingCacheImpl.java

```diff
@@ -177,37 +183,20 @@ private synchronized void updateCurrentVersion(TransactionsLockWatchUpdate updat
         Optional<LockWatchVersion> maybeUpdateVersion = updateForTransactions.startTsToSequence().values().stream()
                 .max(Comparator.comparingLong(LockWatchVersion::version));
 
-        maybeUpdateVersion.ifPresent(updateVersion -> {
-            if (firstUpdateAfterCreation()) {
-                currentVersion = maybeUpdateVersion;
-            } else {
-                LockWatchVersion current = currentVersion.get();
-                if (leaderElection(current, updateVersion)) {
-                    clearCache();
-                    currentVersion = maybeUpdateVersion;
-                } else if (newUpdate(current, updateVersion)) {
-                    currentVersion = maybeUpdateVersion;
-                }
-            }
```

Comment:
We don't need this logic, since we just clear the version on `clearCache` being true from the update anyway.